src/Componets/SelectCategory/SelectCategory.tsx

```diff
@@ -4,22 +4,29 @@ import style from './SelectCategory.module.scss';
 
 type Props = {
     value: string,
-    onChange: Dispatch<SetStateAction<string>>,
+    onChange: Dispatch<SetStateAction<string[]>>,
+    selected: string[];
     options: string[],
     placeholder: string
 }
 function Select({
-  value, onChange, options, placeholder,
+  value, onChange, options, placeholder, selected,
 }: Props) {
```

Comment:
i'm not sure how is it with React, but consider to use some more specific name for props, ex. "SelectProps"